src/types/messages/components/message_component_types.ts

```diff
@@ -0,0 +1,6 @@
+export enum MessageComponentTypes {
```

Comment:
```suggestion
// TODO: add docs link
export enum DiscordMessageComponentTypes {
```
Note: Add an alias named MessageComponentTypes at the bottom